tests/components/wallbox/test_config_flow.py

```diff
@@ -162,3 +163,83 @@ async def test_form_validate_input(hass):
 
     assert result2["title"] == "Wallbox Portal"
     assert result2["data"]["station"] == "12345"
+
+
+async def test_form_reauth(hass):
+    """Test we handle reauth flow."""
+    await setup_integration(hass)
+    assert entry.state == config_entries.ConfigEntryState.LOADED
+
+    with requests_mock.Mocker() as mock_request:
+        mock_request.get(
+            "https://api.wall-box.com/auth/token/user",
+            text='{"jwt":"fakekeyhere","user_id":12345,"ttl":145656758,"error":false,"status":200}',
+            status_code=200,
+        )
+        mock_request.get(
+            "https://api.wall-box.com/chargers/status/12345",
+            json=test_response,
+            status_code=200,
+        )
+
```

Comment:
For a future PR:

Generally we prefer to mock the library calls themselves and test the api in the library.

This avoids having to update both sets of tests (the lib and Home Assistant) when the endpoint or API changes.

Reply:
OK, I will look into this